Tests/Functional/Hooks/InitializeStdWrapTest.php

```diff
@@ -11,54 +11,32 @@
 namespace TYPO3\CMS\Fluid\Tests\Functional\Hooks;
 
 use JWeiland\Jwtools2\Hooks\InitializeStdWrap;
-use Nimut\TestingFramework\TestCase\FunctionalTestCase;
-use Prophecy\PhpUnit\ProphecyTrait;
-use Prophecy\Prophecy\ObjectProphecy;
+use PHPUnit\Framework\MockObject\MockObject;
 use TYPO3\CMS\Core\Configuration\ExtensionConfiguration;
 use TYPO3\CMS\Frontend\ContentObject\ContentObjectRenderer;
+use TYPO3\TestingFramework\Core\Functional\FunctionalTestCase;
 
 /**
  * Test case
  */
 class InitializeStdWrapTest extends FunctionalTestCase
 {
-    use ProphecyTrait;
+    protected InitializeStdWrap $subject;
 
-    /**
-     * @var InitializeStdWrap
-     */
-    protected $subject;
-
-    /**
-     * @var ExtensionConfiguration|ObjectProphecy
-     */
-    protected $extensionConfigurationProphecy;
+    protected ExtensionConfiguration | MockObject $extensionConfigurationMock;
 
-    /**
-     * @var array
-     */
-    protected $testExtensionsToLoad = [
-        'typo3conf/ext/jwtools2',
+    protected array $testExtensionsToLoad = [
+        'jweiland/jwtools2',
     ];
 
     protected function setUp(): void
     {
         parent::setUp();
 
-        $this->extensionConfigurationProphecy = $this->prophesize(ExtensionConfiguration::class);
+        $this->extensionConfigurationMock = $this->getAccessibleMock(ExtensionConfiguration::class);
```

Comment:
A normal `getMock` or `createMock` should be enough as you don't make use of all the specific _set methods